Python/Setup/swix/swix.generate.targets

```diff
@@ -49,10 +49,6 @@ package name=$([System.IO.Path]::GetFileNameWithoutExtension($(OutputName))) ver
 <WriteLinesToFile File="$(SwrFile).tmp" Lines="vs.dependencies
 $(_Dependencies)
 " Overwrite="no" Encoding="ASCII" Condition="$(_Dependencies) != ''" />
-
-    <ItemGroup>
-      <Package Include="$(SwrFile)" />
-    </ItemGroup>
   </Target>
 
   <Target Name="_GenerateSwrFromSources" Condition="@(SwrSource) != ''">
```

Comment:
Except then you can't deploy files into the MSBuild, ProjectTemplates or ItemTemplates directories, and you can't guarantee a stable install directory. So this works out better even though it's a little more work. Most of it is generated from the classic VSIX layout though ;)

---
In reply to: [69643738](https://github.com/Microsoft/PTVS/pull/1396#discussion_r69643738) [](ancestors = 69643738)